pkgs/development/python-modules/pyu2f/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, fetchPypi, buildPythonPackage,
+  six, mock, pyfakefs, unittest2
+}:
+
+buildPythonPackage rec {
+  pname = "pyu2f";
+  version = "0.1.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "12d1260ccd7cfd7a987fc1ad84667770efd6b422cec47fe8eda18d5d52ff6bd3";
+  };
+
+  propagatedBuildInputs = [ six ];
+
+  buildInputs = [ six mock pyfakefs unittest2 ];
+
+  # the tests fail with weird error when OS is detected as Windows
+  doCheck = false;
+
+  meta = {
+    description = "U2F host library for interacting with a U2F device over USB.";
+    homepage = https://github.com/google/pyu2f/;
+    license = lib.licenses.apache;
+    maintainers = with lib.maintainers; [ prusnak ];
```

```suggestion
    maintainers = with maintainers; [ prusnak ];
```
Because https://github.com/NixOS/nixpkgs/pull/52590/files#r243412634